src/ontology/edits/oeo-model.omn

```diff
@@ -3142,8 +3162,12 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652
 
 reclassify as data descriptor
 issue: https://github.com/OpenEnergyPlatform/ontology/issues/1536
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1853",
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A time stamp is a data descriptor that is used to describe a zero-dimensional temporal region, or a data item that is related to a zero-dimensional temporal region, e.g. a time stamped measurement datum.",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1853
+
+update definition
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1901
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1944",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A time stamp is a data descriptor that is used to describe a temporal region, or a data item that is related to a temporal region, e.g. a time stamped measurement datum. If the time stamp refers to a time step/one-dimensional temporal region, the indication of a time stamp alignment is required",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A time stamp is a data descriptor that is used to describe either a temporal region or a data item that is related to a temporal region, e.g. a time stamped measurement datum. If the time stamp refers to a time step/one-dimensional temporal region, the indication of a time stamp alignment is required",
```
I added either. Else it sounded like "a time stamp is a data descriptor or a data item", which would hurt monohierarchy.